stories/cosmoz-input.stories.js

```diff
@@ -46,16 +46,27 @@ export const contour = () => html`
 	<style>
 		cosmoz-input {
 			--cosmoz-input-border-radius: 4px;
-			--cosmoz-input-padding: 8px;
+			--cosmoz-input-padding: 10px;
 			--cosmoz-input-label-width: auto;
 			--cosmoz-input-no-placeholder-label-bg: white;
+			--cosmoz-input-floating-label-bg: transparent;
```

Comment:
Isn't the default value for `--cosmoz-input-floating-label-bg` already transparent?

Reply:
It is set to the value of `var(--bg)`, which is set in turn to the value of `var(--cosmoz-input-background)`, which here is set to `white`. Either way, it's a repetition, I'll drop it (and rename it? Since it would not be public?)

Reply:
On the other hand - do we want the label background to be styled independently from the rest?

Reply:
This is just the story. You should keep things clean and only define the vars that area actually needed to achieve the design. Maybe you also want to clean up `--cosmoz-input-no-placeholder-label-bg`, `--cosmoz-input-label-width` and `--cosmoz-input-background`.